packages/cli/src/util/getHttpOperations.ts

```diff
@@ -0,0 +1,43 @@
+import { transformOas2Operation, transformOas3Operation } from '@stoplight/http-spec';
+import { parse } from '@stoplight/yaml';
+import axios from 'axios';
+import * as fs from 'fs';
+import { defaults, flatten, get, keys, map, uniq } from 'lodash';
+import { EOL } from 'os';
+import { httpAndFileResolver } from '../resolvers/http-and-file';
+
+export default async function getHttpOperations(spec: string) {
+  const fileContent = spec.match(/^https?:\/\//)
+    ? (await axios.get(spec)).data
+    : fs.readFileSync(spec, { encoding: 'utf8' });
+  const parsedContent = parse(fileContent);
+  const { result: resolvedContent, errors } = await httpAndFileResolver.resolve(parsedContent);
+
+  if (errors.length) {
+    const uniqueErrors = uniq(errors.map(error => error.message)).join(EOL);
+    throw new Error(uniqueErrors);
+  }
+
+  const isOas2 = get(parsedContent, 'swagger');
+
+  const transformOperationFn = isOas2 ? transformOas2Operation : transformOas3Operation;
+
+  const paths = keys(get(resolvedContent, 'paths'));
+  const methods = ['get', 'post', 'put', 'delete', 'options', 'head', 'patch', 'trace'];
+  return flatten(
+    map(paths, path =>
+      keys(get(resolvedContent, ['paths', path]))
+        .filter(k => methods.includes(k))
+        .map(method =>
+          defaults(
+            transformOperationFn({
+              document: resolvedContent,
+              path,
+              method,
+            }),
+            { servers: resolvedContent.servers },
+          ),
```

Comment:
@marbemac Is this ok?

Reply:
@XVincentX why are you providing `servers`? `transformOperationFn` should handle it for you IIRC.

Reply:
It's not happening; I agree that's weird, but I've been checking the code and effectively such function is not doing anything with regards of the `servers`. I'll try to dig into more — in meantime maybe @marbemac knows what's going on here?

Reply:
You were right, the server transformation is happening. I removed the code. @chris-miaskowski